recipes/hdf5/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.8.16" %}
+
+package:
+  name: hdf5
+  version: {{ version }}
+
+source:
+  fn: hdf5-{{version}}.tar.gz
+  url: http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-{{version}}.tar.gz
+  md5: b8ed9a36ae142317f88b0c7ef4b9c618
+  patches:
+    - test_Makefile.in.patch
```

Comment:
Yep. Slightly concerned - maybe we should have an environment variable which allows us to turn stuff on if we are ```conda-forge``` building stuff. Like:

```
patches:
    - test_Makefile.in.patch  # [os.environ.get(conda-forge)]
```